.eslintignore

```diff
@@ -0,0 +1 @@
+apps/site/*
```

Comment:
do we need to ignore the eslint here?

Reply:
yes, next have the own eslint config, so when we move the core to here, we'll create our config